src/containers/my-cooperations/cooperation-activities-list/CooperationActivitiesList.tsx

```diff
@@ -36,10 +36,22 @@ const CooperationActivitiesList = () => {
     setSectionsData(data.sections)
   }
 
-  const addNewSection = () => {
+  const addNewSection = (index = null) => {
     const newSectionData = { ...sectionInitialData }
     newSectionData.id = Date.now().toString()
-    setSectionsData([...data.sections, newSectionData])
+    let newSections = [...data.sections]
+
```

Comment:
maybe try next approach with using const instead of let or maybe you will have better solution:
const newSections = index !== null
  ? [
      ...data.sections.slice(0, index),
      newSectionData,
      ...data.sections.slice(index)
    ]
  : [...data.sections, newSectionData];